lib/features/children/family_goal/widgets/family_goal_creation_stepper.dart

```diff
@@ -0,0 +1,141 @@
+                      Expanded(
+                        child: Container(
+                          width: double.maxFinite,
+                          height: 4,
+                          color: currentStep.index >
+                                  FamilyGoalCreationSteps.cause.index
+                              ? AppTheme.givtLightGreen
+                              : AppTheme.givtLightGray,
+                        ),
+                      ),
+                      Expanded(
+                        child: Container(
+                          width: double.maxFinite,
+                          height: 4,
+                          color: currentStep.index >=
+                                  FamilyGoalCreationSteps.confirmation.index
+                              ? AppTheme.givtLightGreen
+                              : AppTheme.givtLightGray,
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    CircleAvatar(
+                      radius: 6.5,
+                      backgroundColor: currentStep.index >
+                              FamilyGoalCreationSteps.overview.index
+                          ? AppTheme.givtLightGreen
+                          : AppTheme.familyGoalStepperGray,
+                    ),
+                    CircleAvatar(
+                      radius: 6.5,
+                      backgroundColor: currentStep.index >
+                              FamilyGoalCreationSteps.cause.index
+                          ? AppTheme.givtLightGreen
+                          : AppTheme.familyGoalStepperGray,
+                    ),
+                    Icon(
+                      FontAwesomeIcons.flagCheckered,
+                      color: currentStep.index >=
+                              FamilyGoalCreationSteps.confirmation.index
+                          ? AppTheme.givtLightGreen
+                          : AppTheme.familyGoalStepperGray,
+                    ),
+                    // Text(
+                    //   flagSwallowtail,
+                    //   style: TextStyle(
+                    //     fontFamily: 'FontAwesome',
+                    //     color: currentStep.index >=
+                    //             FamilyGoalCreationSteps.confirmation.index
+                    //         ? AppTheme.givtLightGreen
+                    //         : AppTheme.familyGoalStepperGray,
+                    //     fontSize: 20,
+                    //   ),
+                    // ),
+                  ],
+                ),
+              ],
+            ),
+          ),
+          const SizedBox(height: 5),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Text(
+                //TODO: POEditor
+                '1. Cause',
```

Comment:
Should we do POEditor before a merge, or deal with this later?

Reply:
I'm going to apply POEditor later on when the entire feature will be finished